docs/spfx/web-parts/guidance/build-custom-property-pane-controls.md

```diff
@@ -0,0 +1,827 @@
+
+In the code editor open the **src/webparts/listItems/ListItemsWebPartManifest.json** file. Replace the default **description** property with a new property called `list`.
+
+![Web part manifest with the 'list' web part property highlighted](../../../../images/custom-property-pane-control-list-property-web-part-manifest.png)
+
+Next, open the **src/webparts/listItems/IListItemsWebPartProps.ts** file and replace its contents with:
+
+```ts
+export interface IListItemsWebPartProps {
+  list: string;
+}
+```
+
+In the **src/webparts/listItems/ListItemsWebPart.ts** file, change the **render** method to:
+
+```ts
+export default class ListItemsWebPart extends BaseClientSideWebPart<IListItemsWebPartProps> {
+  // ...
+  public render(): void {
+    const element: React.ReactElement<IListItemsProps> = React.createElement(ListItems, {
+      list: this.properties.list
+    });
+
+    ReactDom.render(element, this.domElement);
+  }
+  // ...
+}
+```
+
+Update the **propertyPaneSettings** getter to:
+
+```ts
+export default class ListItemsWebPart extends BaseClientSideWebPart<IListItemsWebPartProps> {
+  // ...
+  protected get propertyPaneSettings(): IPropertyPaneSettings {
+    return {
+      pages: [
+        {
+          header: {
+            description: strings.PropertyPaneDescription
+          },
+          groups: [
+            {
+              groupName: strings.BasicGroupName,
+              groupFields: [
+                PropertyPaneTextField('list', {
+                  label: strings.ListFieldLabel
+                })
+              ]
+            }
+          ]
+        }
+      ]
+    };
+  }
+  // ...
+}
+```
+
+In the **src/webparts/listItems/loc/mystrings.d.ts** file change the **IListItemsStrings** interface to:
+
+```ts
+declare interface IListItemsStrings {
+  PropertyPaneDescription: string;
+  BasicGroupName: string;
+  ListFieldLabel: string;
+}
+```
+
+In the **src/webparts/listItems/loc/en-us.js** file add the missing definition for the **ListFieldLabel** string:
+
+```js
+define([], function() {
+  return {
+    "PropertyPaneDescription": "Description",
+    "BasicGroupName": "Group Name",
+    "ListFieldLabel": "List"
+  }
+});
+```
+
+In the **src/webparts/listItems/components/ListItems.tsx** file change the contents of the **render** method to
+
+```tsx
+export default class ListItems extends React.Component<IListItemsProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <div className={styles.listItems}>
+        <div className={styles.container}>
+          <div className={css('ms-Grid-row ms-bgColor-themeDark ms-fontColor-white', styles.row)}>
+            <div className='ms-Grid-col ms-u-lg10 ms-u-xl8 ms-u-xlPush2 ms-u-lgPush1'>
+              <span className='ms-font-xl ms-fontColor-white'>
+                Welcome to SharePoint!
+              </span>
+              <p className='ms-font-l ms-fontColor-white'>
+                Customize SharePoint experiences using Web Parts.
```

Comment:
These are standard strings the way they are being provided by the template. From the tutorial point of view I think it would be good to avoid any changes unrelated to the tutorial to make it clear what's necessary to accomplish the given task versus what's a general good practice.